app/src/pages/trace/TracePage.tsx

```diff
@@ -181,59 +181,57 @@ export function TracePage() {
       query TracePageQuery($traceId: ID!, $id: GlobalID!) {
         project: node(id: $id) {
           ... on Project {
-            spans(
-              traceIds: [$traceId]
-              sort: { col: startTime, dir: asc }
-              first: 1000
-            ) {
-              edges {
-                span: node {
-                  context {
-                    spanId
-                  }
-                  name
-                  spanKind
-                  statusCode: propagatedStatusCode
-                  statusMessage
-                  startTime
-                  parentId
-                  latencyMs
-                  tokenCountTotal
-                  tokenCountPrompt
-                  tokenCountCompletion
-                  input {
-                    value
-                    mimeType
-                  }
-                  output {
-                    value
-                    mimeType
-                  }
-                  attributes
-                  events {
-                    name
-                    message
-                    timestamp
-                  }
-                  spanEvaluations {
-                    name
-                    label
-                    score
-                  }
-                  documentRetrievalMetrics {
-                    evaluationName
-                    ndcg
-                    precision
-                    hit
-                  }
-                  documentEvaluations {
-                    documentPosition
+            trace(traceId: $traceId) {
+              spans {
```

Comment:
You forgot the pagination 1000 I think. Probably healthy to have a default so you don't over-fetch by default